+vis/patch.m

```diff
@@ -1,60 +1,63 @@
 function elem = patch(t, shape)
 %VIS.PATCH Returns a Signals patch stimulus defining a texture of a shape
-%  Produces a visual stimulus of a shape.
+%  Produces a visual element for parameterizing the presentation of a
+%  shape.
 %
 %  Inputs:
-%    t - any signal with which to derive the network ID.  By convetion we
-%      use the t signal
-%    shape - char array defining what shape to show. Options are rectangle 
-%      (default), circle and cross
+%    't' - The "time" signal. Used to obtain the Signals network ID.
+%      (Could be any signal within the network - 't' is chosen by
+%      convention).
+%    'shape' - char array defining what shape to show. Options are 
+%     'rectangle' (default), 'circle' and 'cross'.
 %    
 %  Outputs:
-%    elem - a subscriptable signal containing paramter fields for the
-%      stimulus along with the processed texture layer.  Any parameter may
-%      be a signal.  
+%    'elem' - a subscriptable signal containing fields which parametrize
+%      the stimulus, and a field containing the processed texture layer. 
+%      Currently, any of the fields but 'dims' for 'circle' may be a signal.
```

Comment:
I saw you made changes to circLayer and added some more functions to Signals, are you sure this is still the case?

Reply:
Yeah, still doesn't work due to the way MATLAB calls 'meshgrid'